lib/bundle/commands/check.rb

```diff
@@ -14,50 +16,66 @@ def reset!
         Bundle::BrewServices.reset!
       end
 
+      def exit_on_first_error?
+        Bundle::Checker.exit_on_first_error?
+      end
+
+      def output_errors?
+        ARGV.include?("--verbose")
+      end
+
       def run
-        if any_taps_to_tap? ||
-           any_casks_to_install? ||
-           any_apps_to_install? ||
-           any_formulae_to_install? ||
-           any_formulae_to_start?
+        @dsl ||= Bundle::Dsl.new(Bundle.brewfile)
+
+        checks = {
+          taps_to_tap: "Taps",
+          casks_to_install: "Casks",
+          apps_to_install: "Apps",
+          formulae_to_install: "Formulae",
+        }
+        check_method_names = checks.keys
+
+        completed_checks = []
+        errors = []
+        enumerator = exit_on_first_error? ? :any? : :each
+
+        work_to_be_done = check_method_names.send(enumerator) do |check_method|
```

Comment:
When this is `.send(:each)` it's not going to return `any_errors` but it'll just be `check_method_names`. That's the cause of the test failure.